server/prisma/schema.prisma

```diff
@@ -0,0 +1,18 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
```

Comment:
A barebone user table usually has additional fields alongside what you have currently such as:

  password  (stores the password of the user)
  createdAt (when the users account was created)
  updatedAt (if the user updates there profile for example this is a timestamp to show when they updated their information)